frontend/packages/console-app/src/components/quick-starts/loader/QuickStartsLoader.tsx

```diff
@@ -21,23 +21,30 @@ const QuickStartsLoader: React.FC<QuickStartsLoaderProps> = ({ children }) => {
   }, [quickStarts, quickStartsLoaded]);
 
   const [allowedQuickStarts, setAllowedQuickStarts] = React.useState<QuickStart[]>([]);
-  const [permissionsLoaded, setPermissionsLoaded] = React.useState<boolean>(false);
-  const permissionChecks = React.useRef<{ [name: string]: boolean }>({});
+  const [permissionsResolved, setPermissionsResolved] = React.useState<boolean>(false);
+  const permissionCheckResults = React.useRef<{ [name: string]: boolean }>({});
 
   const handlePermissionCheck = React.useCallback(
     (quickStart, hasPermission) => {
-      permissionChecks.current[quickStart.metadata.name] = hasPermission;
-      if (Object.keys(permissionChecks.current).length === enabledQuickstarts.length) {
+      permissionCheckResults.current[quickStart.metadata.name] = hasPermission;
+      if (Object.keys(permissionCheckResults.current).length === enabledQuickstarts.length) {
         const filteredQuickStarts = enabledQuickstarts.filter(
-          (quickstart) => permissionChecks.current[quickstart.metadata.name],
+          (quickstart) => permissionCheckResults.current[quickstart.metadata.name],
         );
         setAllowedQuickStarts(filteredQuickStarts);
-        setPermissionsLoaded(true);
+        setPermissionsResolved(true);
       }
     },
     [enabledQuickstarts],
   );
 
+  // Show content (see QuickStartDrawer) when
+  // 1. Quick starts are loaded or couldn't loaded (error)
+  // 2. When there is no quick start or all permission checks are resolved.
+  const loaded =
+    (quickStartsLoaded || !!quickStartsError) &&
```

Comment:
Added this `!!quickStartError` to ensure that the children in QuickStartDrawer are rendered when an error happen.